docs/multi-module-testing.md

```diff
@@ -103,6 +103,15 @@ You may want to exclude a library module from testing when using Fulladle. You c
       enabled.set(false)
     }
     ```
+
+
+### Overriding root-level config
+Flank offers two different ways to specify APKs for testing, one is the *root-level* configuration (under the `gcloud` namespace) and the other is as an `additional-app-test-apks` (under the `flank` namespace).
```

Comment:
I appreciate the insight here but this seems like a implementation detail of flank and not relevant to fladle.